src/styles/_exports.scss

```diff
@@ -0,0 +1,25 @@
+@import "colors";
```

Comment:
is this used? since it seems like we re-define all the colors here anyway

Reply:
I'm dumb; I was just following the example on StackOverflow because my mind was so broken from trying to style a third-party component with SCSS 😆 I'm fixing it, will push up after I merge in some other files and make sure the branch works.